domains/yohan.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "YohanReddy",
+           "email": "yohanreddy1@gmail.com",
+           "discord": "711835508946239582"
+        },
+    
+        "record": {
+            "TXT": "_vercel.is-a.dev"
```

Comment:
That's not a valid domain of ours. We do not support Vercel either way. TXT records by themselves do not do anything.